tests/pages/staff-dashboard.test.tsx

```
@@ -263,12 +264,69 @@ describe("Feature: Dashboard do painel interno", () => {
+      });
+      fireEvent.click(screen.getByText("Cancelar"));
+
+      await waitFor(() => {
+        expect(screen.getByText("Retorno cancelado")).toBeInTheDocument();
+      });
+    });
+
+    it("Dado falha ao concluir retorno, Quando acionado, Então exibe toast de erro", async () => {
+      vi.stubGlobal(
+        "fetch",
+        vi.fn(async (input: RequestInfo | URL, init?: RequestInit) => {
```

ESLint

[error] Async arrow function has a complexity of 11. Maximum allowed is 10.

(complexity)